cmd/kubelet/app/options/options.go

```diff
@@ -476,6 +476,8 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.MarkDeprecated("iptables-drop-bit", "This flag has no effect and will be removed in a future version.")
 	fs.StringVar(&c.ContainerLogMaxSize, "container-log-max-size", c.ContainerLogMaxSize, "<Warning: Beta feature> Set the maximum size (e.g. 10Mi) of container log file before it is rotated.")
 	fs.Int32Var(&c.ContainerLogMaxFiles, "container-log-max-files", c.ContainerLogMaxFiles, "<Warning: Beta feature> Set the maximum number of container log files that can be present for a container. The number must be >= 2.")
+	fs.Int32Var(&c.ContainerLogMaxWorkers, "container-log-max-workers", c.ContainerLogMaxWorkers, "<Warning: Beta feature> Set the maximum number of concurrent workers to spawn for performing the log rotation. Set the value to 0 to disable concurrent rotation. Defaults to 0")
+	fs.DurationVar(&c.ContainerLogMonitorInterval.Duration, "container-log-monitor-interval", c.ContainerLogMonitorInterval.Duration, "<Warning: Beta feature> Set the interval at which kubelet will monitor the logs for performing rotation. Defaults to 10s.")
```

Comment:
do we have to add the flags here? can these *just* be settable via the config file?

```
		// All KubeletConfiguration flags are now deprecated, and any new flags that point to
		// KubeletConfiguration fields are deprecated-on-creation.
```

Reply:
I am fine to retain these as config file only flags. That will be inline with that comment. Let me make the changes